openexr-sys/c_wrapper/cexr.cpp

```diff
@@ -127,7 +127,6 @@ int CEXR_Header_get_channel(const CEXR_Header *header, const char name[], const
         *out = channel_ptr;
         return 0;
     } else {
-        *err_out = "no channel found with that name";
         return 1;
```

Comment:
Ah, I guess not, no.  I was keeping this and the FrameBuffer get_channel superficially consistent, but since their semantics are different anyway (one gives you a pointer, the other gives you a value) it probably actually makes it less confusing if I don't make them superficially consistent.  I'll change that in master.